main.py

```diff
@@ -1,51 +1,30 @@
-
-MAJOR_COLORS = ['White', 'Red', 'Black', 'Yellow', 'Violet']
-MINOR_COLORS = ["Blue", "Orange", "Green", "Brown", "Slate"]
-
+from colorCode import ColorCode
```

Comment:
main.py contains production code and test code. It contains more processing code than it should, or else the file name should be renamed to suit the content.
Or maybe test code can be maintained separately in a different file.

Reply:
Issue is solved